api/app/jobs/common_model_fetchers.py

```diff
@@ -169,10 +169,10 @@ class ModelValueProcessor:
     """ Iterate through model runs that have completed, and calculate the interpolated weather predictions.
     """
 
-    def __init__(self, session):
+    def __init__(self, session, station_source: StationSourceEnum = None):
```

Comment:
What's the difference between `None` and `UNSPECIFIED` for station source? Can we simplify?

Reply:
Good catch...should be `UNSPECIFIED`

Fixed in https://github.com/bcgov/wps/pull/2858/commits/24aa59521fb7628be4edb52f24560ccc36920504